oceans-nadia.com,"HTTPSConnectionPool(host='oceans-nadia.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oceans-nadia.com', port=443): Read timed out. (read timeout=2)"
ocn.ne.jp,"HTTPSConnectionPool(host='www.ocn.ne.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.ocn.ne.jp', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
oeticket.com,"HTTPSConnectionPool(host='oeticket.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde32e7fa0>, 'Connection to oeticket.com timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.oeticket.com', port=80): Read timed out. (read timeout=2)"
ofertolino.ro,"HTTPSConnectionPool(host='ofertolino.ro', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='ofertolino.ro', port=443): Read timed out. (read timeout=2)"
offen.net,"HTTPSConnectionPool(host='offen.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.offen.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
office1.bg,"HTTPSConnectionPool(host='www.office1.bg', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.office1.bg', port=443): Read timed out. (read timeout=2)"
officedepot.co.cr,"HTTPSConnectionPool(host='officedepot.co.cr', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='officedepot.co.cr', port=80): Read timed out. (read timeout=2)"
officedepot.com.sv,"HTTPSConnectionPool(host='officedepot.com.sv', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde2077ee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='officedepot.com.sv', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde4af9b20>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
offipalsta.com,"HTTPSConnectionPool(host='offipalsta.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='offipalsta.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
ofppt.ma,"HTTPSConnectionPool(host='ofppt.ma', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde4f5b9d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ofppt.ma', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde2205d90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ogledalofirme.com,"HTTPSConnectionPool(host='ogledalofirme.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ogledalofirme.com' doesn't match 'sadrzaj.ogledalofirme.com'"")))","HTTPSConnectionPool(host='www.ogledalofirme.com', port=443): Read timed out. (read timeout=2)"
oic.or.th,"HTTPSConnectionPool(host='oic.or.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde2eeeca0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='oic.or.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde1e3b730>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
oj.gob.gt,"HTTPSConnectionPool(host='oj.gob.gt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde130a8e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='oj.gob.gt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde1dd56a0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
okanhastanesi.com.tr,"HTTPSConnectionPool(host='okanhastanesi.com.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='okanhastanesi.com.tr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
oleole.pl,"HTTPSConnectionPool(host='www.oleole.pl', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.oleole.pl', port=443): Read timed out. (read timeout=2)"
oliberal.com,"HTTPSConnectionPool(host='www.oliberal.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oliberal.com', port=443): Read timed out. (read timeout=2)"
olivemagazine.gr,"HTTPSConnectionPool(host='olivemagazine.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))","HTTPSConnectionPool(host='olivemagazine.gr', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1124)')))"
olnafu.ru,"HTTPSConnectionPool(host='olnafu.ru', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='olnafu.ru', port=443): Read timed out. (read timeout=2)"
olx.co.id,"HTTPSConnectionPool(host='olx.co.id', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde0d9b190>, 'Connection to olx.co.id timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.olx.co.id', port=80): Read timed out. (read timeout=2)"
olx.co.za,"HTTPSConnectionPool(host='olx.co.za', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde3c29d00>, 'Connection to olx.co.za timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.olx.co.za', port=443): Read timed out. (read timeout=2)"
olx.com.ec,"HTTPSConnectionPool(host='olx.com.ec', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdddf7e9d30>, 'Connection to olx.com.ec timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.olx.com.ec', port=443): Read timed out. (read timeout=2)"
olx.com.eg,"HTTPSConnectionPool(host='www.olx.com.eg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdddf6d9a30>, 'Connection to www.olx.com.eg timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.olx.com.eg', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdddf544fd0>, 'Connection to www.olx.com.eg timed out. (connect timeout=2)'))"
olx.com.gt,"HTTPSConnectionPool(host='olx.com.gt', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdddf6d9cd0>, 'Connection to olx.com.gt timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='www.encuentra24.com', port=443): Read timed out. (read timeout=2)"
olx.in,"HTTPSConnectionPool(host='olx.in', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdddf736f40>, 'Connection to olx.in timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.olx.in', port=80): Read timed out. (read timeout=2)"
omelhordobairro.com,"HTTPSConnectionPool(host='omelhordobairro.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.omelhordobairro.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
omni7.jp,"HTTPSConnectionPool(host='omni7.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde57408e0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='omni7.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdddf52be80>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
omron.co.jp,"HTTPSConnectionPool(host='omron.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde53aaaf0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='omron.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde00d2a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
oncyprus.com,"HTTPSConnectionPool(host='www.oncyprus.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.oncyprus.com', port=443): Read timed out. (read timeout=2)"
onep.go.th,"HTTPSConnectionPool(host='onep.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fddddc3db50>, 'Connection to onep.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='onep.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde3d0ebb0>, 'Connection to onep.go.th timed out. (connect timeout=2)'))"
onepagefree.com,"HTTPSConnectionPool(host='onepagefree.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: UNSUPPORTED_PROTOCOL] unsupported protocol (_ssl.c:1124)')))","HTTPConnectionPool(host='onepagefree.com', port=80): Read timed out. (read timeout=2)"
onesearch.id,"HTTPSConnectionPool(host='onesearch.id', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='onesearch.id', port=443): Read timed out. (read timeout=2)"
onko-i.si,"HTTPSConnectionPool(host='onko-i.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fddddc76430>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='onko-i.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde44156d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
onlineexpo.com,"HTTPSConnectionPool(host='onlineexpo.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='onlineexpo.com', port=443): Read timed out. (read timeout=2)"
oocities.org,"HTTPSConnectionPool(host='oocities.org', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='oocities.org', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde2a73e20>, 'Connection to oocities.org timed out. (connect timeout=2)'))"
opais.co.ao,"HTTPSConnectionPool(host='opais.co.ao', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='opais.co.ao', port=443): Read timed out. (read timeout=2)"
openingsuren.info,"HTTPSConnectionPool(host='www.openingsuren.info', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.openingsuren.info', port=443): Read timed out. (read timeout=2)"
opiniodajnia.pl,"HTTPSConnectionPool(host='opiniodajnia.pl', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='opiniodajnia.pl', port=80): Read timed out. (read timeout=2)"
oppo.com,"HTTPSConnectionPool(host='oppo.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde1785370>, 'Connection to oppo.com timed out. (connect timeout=2)'))","HTTPSConnectionPool(host='oppo.com', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdddc45a640>, 'Connection to oppo.com timed out. (connect timeout=2)'))"
opsmoac.go.th,"HTTPSConnectionPool(host='opsmoac.go.th', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde016cfa0>, 'Connection to opsmoac.go.th timed out. (connect timeout=2)'))","HTTPConnectionPool(host='opsmoac.go.th', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde0132a90>, 'Connection to opsmoac.go.th timed out. (connect timeout=2)'))"
optus.com.au,"HTTPSConnectionPool(host='www.optus.com.au', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='www.optus.com.au', port=80): Read timed out. (read timeout=2)"
ordenjuridico.gob.mx,"HTTPSConnectionPool(host='ordenjuridico.gob.mx', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde30b88b0>, 'Connection to ordenjuridico.gob.mx timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ordenjuridico.gob.mx', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde3013940>, 'Connection to ordenjuridico.gob.mx timed out. (connect timeout=2)'))"
ore.edu.pl,"HTTPSConnectionPool(host='ore.edu.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='ore.edu.pl', port=443): Max retries exceeded with url: // (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
orebro.se,"HTTPSConnectionPool(host='www.orebro.se', port=443): Read timed out.","HTTPSConnectionPool(host='www.orebro.se', port=443): Read timed out."
organiseereentopevent.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
orst.go.th,"HTTPSConnectionPool(host='orst.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde272b490>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='orst.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde425d610>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
osaka-u.ac.jp,"HTTPSConnectionPool(host='osaka-u.ac.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde4a30430>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='osaka-u.ac.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde54467f0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
osaka.lg.jp,"HTTPSConnectionPool(host='osaka.lg.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde5102df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='osaka.lg.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde31eb760>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
ose.com.uy,"HTTPSConnectionPool(host='ose.com.uy', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLEOFError(8, 'EOF occurred in violation of protocol (_ssl.c:1124)')))","HTTPConnectionPool(host='ose.com.uy', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde3194880>, 'Connection to ose.com.uy timed out. (connect timeout=2)'))"
osloskolen.no,"HTTPSConnectionPool(host='osloskolen.no', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde2ab3f70>, 'Connection to osloskolen.no timed out. (connect timeout=2)'))","HTTPConnectionPool(host='osloskolen.no', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde22d6370>, 'Connection to osloskolen.no timed out. (connect timeout=2)'))"
othaimmarkets.com,"HTTPSConnectionPool(host='othaimmarkets.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.othaimmarkets.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
otpbanka.hr,"HTTPSConnectionPool(host='otpbanka.hr', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde44da580>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='otpbanka.hr', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde40cdee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
oup.com.pk,"HTTPSConnectionPool(host='oup.com.pk', port=443): Read timed out. (read timeout=2)","HTTPConnectionPool(host='oup.com.pk', port=80): Read timed out. (read timeout=2)"
ous-hf.no,"HTTPSConnectionPool(host='ous-hf.no', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde4a66580>, 'Connection to ous-hf.no timed out. (connect timeout=2)'))","HTTPConnectionPool(host='ous-hf.no', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdddc3c5fd0>, 'Connection to ous-hf.no timed out. (connect timeout=2)'))"
overclockers.ru,"('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))","('Connection aborted.', ConnectionResetError(104, 'Connection reset by peer'))"
overstock.ca,"HTTPSConnectionPool(host='www.overstock.ca', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.overstock.ca', port=443): Read timed out. (read timeout=2)"
overstock.com,"HTTPSConnectionPool(host='www.overstock.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.overstock.com', port=443): Read timed out. (read timeout=2)"
ovt.lv,"HTTPSConnectionPool(host='ovt.lv', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname 'ovt.lv' doesn't match either of '*.datateks.lv', 'datateks.lv'"")))","HTTPConnectionPool(host='ovt.lv', port=80): Read timed out. (read timeout=2)"
owlapps.net,"HTTPSConnectionPool(host='owlapps.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.owlapps.net', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
oxfordlearnersdictionaries.com,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
oxfordreference.com,"HTTPSConnectionPool(host='oxfordreference.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='oxfordreference.com', port=443): Read timed out. (read timeout=2)"
ozs.si,"HTTPSConnectionPool(host='ozs.si', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde0775bb0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='ozs.si', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde41d3340>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
p-world.co.jp,"HTTPSConnectionPool(host='p-world.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde5386a60>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='p-world.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde3fdd070>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
painaidii.com,"HTTPSConnectionPool(host='painaidii.com', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPConnectionPool(host='www.painaidii.com', port=80): Read timed out. (read timeout=2)"
paintandpowerwashct.com,"HTTPSConnectionPool(host='paintandpowerwashct.com', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdddc464610>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='paintandpowerwashct.com', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde28b3d30>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
pais.co.il,"HTTPSConnectionPool(host='pais.co.il', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde130a6a0>, 'Connection to pais.co.il timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pais.co.il', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde05f7a60>, 'Connection to pais.co.il timed out. (connect timeout=2)'))"
paknsaveonline.co.nz,"HTTPSConnectionPool(host='www.paknsaveonline.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.paknsaveonline.co.nz', port=443): Read timed out. (read timeout=2)"
pal-system.co.jp,"HTTPSConnectionPool(host='pal-system.co.jp', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde42ab1c0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='pal-system.co.jp', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde44d9d00>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
palangostiltas.lt,"HTTPSConnectionPool(host='www.palangostiltas.lt', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.palangostiltas.lt', port=443): Read timed out. (read timeout=2)"
palmers.co.nz,"HTTPSConnectionPool(host='www.palmers.co.nz', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.palmers.co.nz', port=443): Read timed out. (read timeout=2)"
palungjit.org,"HTTPSConnectionPool(host='palungjit.org', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='palungjit.org', port=443): Read timed out. (read timeout=2)"
pami.org.ar,"HTTPSConnectionPool(host='pami.org.ar', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdddea27340>, 'Connection to pami.org.ar timed out. (connect timeout=2)'))","HTTPConnectionPool(host='www.pami.org.ar', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde3bbf610>, 'Connection to www.pami.org.ar timed out. (connect timeout=2)'))"
pamiatky.sk,"HTTPSConnectionPool(host='pamiatky.sk', port=443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fdde461a460>, 'Connection to pamiatky.sk timed out. (connect timeout=2)'))","HTTPConnectionPool(host='pamiatky.sk', port=80): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at 0x7fdde3cb89a0>, 'Connection to pamiatky.sk timed out. (connect timeout=2)'))"
panspc.lt,"HTTPSConnectionPool(host='panspc.lt', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdddc217370>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='panspc.lt', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdddbf71160>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
paolohospital.com,"HTTPSConnectionPool(host='paolohospital.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.paolohospital.com', port=443): Read timed out. (read timeout=2)"
paraguay.com,"HTTPSConnectionPool(host='paraguay.com', port=443): Read timed out. (read timeout=2)","HTTPSConnectionPool(host='www.paraguay.com', port=443): Read timed out. (read timeout=2)"
parks.it,"HTTPSConnectionPool(host='parks.it', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde21f9fd0>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='parks.it', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde4b0dee0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
parlament.ch,Exceeded 30 redirects.,Exceeded 30 redirects.
parlament.gov.rs,"HTTPSConnectionPool(host='parlament.gov.rs', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdddbd5cd30>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='parlament.gov.rs', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdde3d5d520>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
parliament.go.th,"HTTPSConnectionPool(host='parliament.go.th', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fdde42fbb50>: Failed to establish a new connection: [Errno -2] Name or service not known'))","HTTPConnectionPool(host='parliament.go.th', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7fdddc2e3df0>: Failed to establish a new connection: [Errno -2] Name or service not known'))"
parp.gov.pl,"HTTPSConnectionPool(host='parp.gov.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='parp.gov.pl', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
partyalert.nl,"('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))","('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))"
pasgo.vn,"HTTPSConnectionPool(host='pasgo.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='pasgo.vn', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
paslaugos24.lt,"HTTPSConnectionPool(host='paslaugos24.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))","HTTPSConnectionPool(host='www.paslaugos24.lt', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1124)')))"
